syntax = "proto3";

package sputnik.pb;

message TokenActivation {
  int32 token = 1;
  bool active = 2; // The new status of the token.
}

message TokenActivationRequest {
  repeated TokenActivation token_activations = 1;
}

message DeleteTokenRequest {
  repeated int32 delete_tokens = 1;
}
